notifications-worker.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/dxe/alc-mobile-api/expo"
+
+	"github.com/dxe/alc-mobile-api/model"
+
+	"github.com/jmoiron/sqlx"
+)
+
+func NotificationsWorker(db *sqlx.DB) {
+	log.Println("Notifications Worker started.")
+
+	tx, err := db.Beginx()
```

Comment:
Don't we need to provide the transaction level? Or is the default adequate?

Reply:
This function basically just executes a "begin" statement. Then when we do the "select ... for update", it seems to lock what it needs to so that nothing else can select it.